### PR TITLE
Support for inline audio

### PR DESCRIPTION
**Pull request by NeonKirill**
Added inline audio support per request

### CODE REVIEW

File: neon_audio/speech.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 from ovos_utils.signal import check_for_signal
 from mycroft_bus_client import Message, MessageBusClient
 
+import librosa
```
Comment (NeonJarbas) [inline neon_audio/speech.py:29]: cant this be done without dragging a new dep?

File: neon_audio/speech.py [hunk 2]
```diff
@@ -60,7 +62,25 @@ def handle_get_tts(message):
...
-            # TODO: Consider including audio bytes here in case path is inaccessible DM
+
+            # TODO: unit testing for this code chunk (Kirill)
+            supported_audio_formats = ['mp3']
+            if message.data.get('return_inline'):
+                for lang in list(responses):
+                    for gender in ('male', 'female'):
+                        audio_format = responses[lang].get(gender, '').split('.')[-1]
+                        if audio_format in supported_audio_formats:
+                            data, sample_rate = librosa.load(responses[lang][gender])
```
[the remark this answers is not in the document]
Comment (NeonDaniel) [reply neon_audio/speech.py:73]: Looks like this was extracted and could be referenced directly here https://github.com/NeonGeckoCom/neon-skill-utils/blob/a9fa0de452004b613ec866a8ab62b107d476e947/neon_utils/file_utils.py#L186

File: neon_audio/speech.py (hunk 1 above)
Comment (NeonDaniel) [reply neon_audio/speech.py:29]: Not sure if there's another method, but it was moved here: https://github.com/NeonGeckoCom/neon-skill-utils/blob/a9fa0de452004b613ec866a8ab62b107d476e947/neon_utils/file_utils.py#L186 .. IIRC, there isn't any pure-python package for audio transcoding; they all seem to pull from `libsndfile` or `ffmpeg`

**Comment by NeonDaniel** [comment]
I think this is replaced by: https://github.com/NeonGeckoCom/neon_audio/commit/7cfd7d0aa2e648e9c8bc79cded2ea7ce13e56f76#diff-527a6e1d4b4073ea5bfd40bc8a1c00c9b256882b14a3da8072d79dc3573015e1R582

If not, please re-open PR

**Pull request closed without merging**